### ISSUE TITLE
Multi-dimensional information for a data point

### ISSUE THREAD

**Issue by GoogleCodeExporter** [josejaner/flot#176]
```
Hello,
i would like to store for a certain data point in a flot graph additional
values. Actually i'am clicking into the graph on acertain data point and
got its x-value. I would also retrieve some more data by doing this (in my
conceptional model the x-value is associated with other values, think of a
plot of temperature (y) vs date (x) and you would also like to know if it
was dry or rainy on the certain date you click).

Thanks a lot.
```

Original issue reported on code.google.com by `mork0...@googlemail.com` on 18 Jun 2009 at 2:46
- Merged into: #121

**Comment by GoogleCodeExporter** [comment]
```
This is currently pending on making data processing a bit smarter. Currently 
it'll
clean up the data input with a hard-coded model. It needs to be a bit more 
flexible.
```

Original comment by `olau%iol...@gtempaccount.com` on 23 Jun 2009 at 5:12
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
this is an important feature that is missing
```

Original comment by `zig...@nullriver.com` on 11 Feb 2012 at 5:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `dnsch...@gmail.com` on 7 May 2012 at 11:08
- Changed state: **Duplicate**
